keyboards/hotdox/keymaps/default/keymap.c

```diff
@@ -0,0 +1,222 @@
+#include QMK_KEYBOARD_H
+#include "debug.h"
```

Comment:
This line is not actually needed, and should be removed. 

```suggestion
```

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.